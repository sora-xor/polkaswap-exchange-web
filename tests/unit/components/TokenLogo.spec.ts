// import omit from 'lodash/fp/omit';
// import Vuex from 'vuex';
// import { shallowMount } from '@vue/test-utils';
// import { api } from '@soramitsu/soraneo-wallet-web';
//
// import TokenLogo from '@/components/TokenLogo.vue';
// import { useDescribe, localVue } from '../../utils';
// import { MOCK_WHITELIST_ITEM, MOCK_TOKEN_LOGO } from '../../mocks/TokenLogoMock';
//
// useDescribe('TokenLogo.vue', TokenLogo, () => {
//   MOCK_TOKEN_LOGO.map((item) =>
//     it(`[${item.title}]: should be rendered correctly`, () => {
//       const store = new Vuex.Store({
//         getters: {
//           whitelist: () => {
//             return { '0x0200000000000000000000000000000000000000000000000000000000000000': MOCK_WHITELIST_ITEM };
//           },
//           whitelistIdsBySymbol: () => api.assets.getWhitelistIdsBySymbol([MOCK_WHITELIST_ITEM]),
//         },
//       });
//       const propsData = omit(['title'], item);
//       const wrapper = shallowMount(TokenLogo, {
//         localVue,
//         store,
//         propsData,
//       });
//       expect(wrapper.element).toMatchSnapshot();
//     })
//   );
// });

describe('Test', () => {
  test('Temporary test', () => {
    expect(true).toEqual(true);
  });
});
