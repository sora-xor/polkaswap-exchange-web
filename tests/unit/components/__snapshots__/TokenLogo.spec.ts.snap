// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TokenLogo.vue [With Big Logo Size]: should be rendered correctly 1`] = `
<span
  class="token-logo token-logo--big"
  style="background-size: 100%; background-image: url(base64_image);"
/>
`;

exports[`TokenLogo.vue [With Default Props]: should be rendered correctly 1`] = `
<span
  class="token-logo token-logo--medium"
/>
`;

exports[`TokenLogo.vue [With Large Logo Size]: should be rendered correctly 1`] = `
<span
  class="token-logo token-logo--large"
  style="background-size: 100%; background-image: url(base64_image);"
/>
`;

exports[`TokenLogo.vue [With Medium (default) Logo Size]: should be rendered correctly 1`] = `
<span
  class="token-logo token-logo--medium"
  style="background-size: 100%; background-image: url(base64_image);"
/>
`;

exports[`TokenLogo.vue [With Mini Logo Size]: should be rendered correctly 1`] = `
<span
  class="token-logo token-logo--mini"
  style="background-size: 100%; background-image: url(base64_image);"
/>
`;

exports[`TokenLogo.vue [With Small Logo Size]: should be rendered correctly 1`] = `
<span
  class="token-logo token-logo--small"
  style="background-size: 100%; background-image: url(base64_image);"
/>
`;

exports[`TokenLogo.vue [With Token Symbol]: should be rendered correctly 1`] = `
<span
  class="token-logo token-logo--medium"
  style="background-size: 100%; background-image: url(base64_image);"
/>
`;
